CodeEdit/Features/AppPreferences/Sections/TerminalPreferences/TerminalPreferencesView.swift

```diff
@@ -41,6 +44,23 @@ struct TerminalPreferencesView: View {
         }
         .frame(width: inputWidth)
     }
+    private var carrotSelector: some View {
+        Picker("Terminal Carrot Style: ", selection: $prefs.preferences.terminal.cursorStyle) {
```

Comment:
Either use `Terminal Caret Style:` or `Terminal Cursor Style:` here

Reply:
Well I guess I’m not eating any CodeEdit carrots 😄